api/rpc/logs/logs.proto

```diff
@@ -0,0 +1,26 @@
+syntax = "proto3";
+
+package logs;
+
+service Logs {
+    rpc Get (GetRequest) returns (GetReply) {
+    }
+}
+
+message LogEntry {
+    string timestamp = 1;
+    string TimeId = 2;
```

Comment:
Please conform to the [protocol buffers style guide](https://developers.google.com/protocol-buffers/docs/style) for naming fields. Use all lowercase and underscores. The language-specific drivers will convert them to to accessors that conform to language (Go) conventions.